awscli/examples/cloudfront/create-invalidation.rst

```diff
@@ -1,5 +1,12 @@
 The following command creates an invalidation for a CloudFront distribution with the ID ``S11A16G5KZMEQD``::
 
+  aws cloudfront create-invalidation --distribution-id S11A16G5KZMEQD \
+    --paths /index.html /error.html
+
+The --paths will automatically generate a random CallerReference every time.
```

Comment:
CallerReference --> ``CallerReference``
If you are going to use the actual parameter name. Same comment applies to the line below.